source/includes/_people.md

```diff
@@ -256,9 +289,19 @@ This endpoint searches for people and do not return any email information. To ge
 
 Parameter | Description | Example
 --------- | ----------- | -----------
-person_titles (optional) | An array of the person's title. Apollo will return results matching ANY of the titles passed in |  ["sales director", "engineer manager"]
-q_organization_domains (optional) | An array of the company domains to search for, joined by the new line character.  | "google.com\nfacebook.com"
+person_titles (optional)                | An array of the person's title. Apollo will return results matching ANY of the titles passed in |  ["sales director", "engineer manager"]
+q_keywords (optional)                   | A string of words over which we want to filter the results | "Tim"
+prospected_by_current_team (optional)   | An array of string booleans defining whether we want models prospected by current team or not. "no" means to look in net new database only, "yes" means to see saved contacts only | ["no"]
+person_locations (optional)             | An array of strings denoting allowed locations of the person | ["California, US", "Minnesota, US"]
+person_seniorities (optional)           | An array of strings denoting the seniorities or levels  | ["senior", "manager"]
+contact_email_status (optional)         | An array of strings to look for people having a set of email statuses | ["verified", "guessed", "unavailable", "bounced", "pending_manual_fulfillment"]
+q_organization_domains (optional)       | An array of the company domains to search for, joined by the new line character.  | "google.com\nfacebook.com"
+organization_locations (optional)       | An array of strings denoting allowed locations of organization headquarters of the person | ["California, US", "Minnesota, US"]
+organization_ids (optional)             | An array of organization ids obtained from companies-search | ["63ff0bc1ff57ba0001e7eXXX"]
+organization_num_employees_ranges (optional)   | An array of intervals to include people belonging in an organization having number of employees in a range | ["1,10", "101,200" ]
 page (optional) | An integer that allows you to paginate through the results  | 1
+per_page (optional) | An integer to load per_page results on a page. Should be in inclusive range [1, 100] | 10
```

Comment:
We are exposing this field which can be easily misused by giving big numbers here. I did not find the validation on the limit of value for per_page. Should we expose this or expose and add validations in backend if it is not present?

Basically, we should not allow the customer to get people more than 100 in a single page

Reply:
We are validating in two places.
1. For endpoints that do not require API Key, we check per-page [here](https://github.com/apolloio/leadgenie/blob/master/lib/rate_limiter.rb/#L368). The default max is 100
2. For APIs using es-searcher and supporting per page, we check in es_searcher validations [here](url)
<img width="891" alt="Screenshot 2023-10-23 at 5 19 55 PM" src="https://github.com/apolloio/apollo-api-docs/assets/54931749/b3bc32f3-6cb6-462e-ac77-c1f1534c8994">

Here, if we have id mode then the limit is 200 else 100. Verifed that for this endpoint, this validation is being conducted. @atul-tw